File: MODERN-ART-ASSOCIATION-OF-KAOHSIUNG/Models/TableModel/Exhibition.cs
namespace MODERN_ART_ASSOCIATION_OF_KAOHSIUNG.Models.TableModel
{
    public class Exhibition
    {
        /// <summary>
        /// 資料表名稱
        /// </summary>
        public string TABLE_NAME { get { return "Exhibition"; } }
        
        /// <summary>
        /// 展覽詳細資料
        /// </summary>
        public string Exhibition_Detail { get; set; }

        /// <summary>
        /// 展覽結束時間
        /// </summary>
        public string Exhibition_Enddatetime { get; set; }

        /// <summary>
        /// 展覽編號
        /// </summary>
        public string Exhibition_ID { get; set; }

        /// <summary>
        /// 其他參展人
        /// </summary>
        public string Exhibition_Otherpeople { get; set; }

        /// <summary>
        /// 展覽圖片
        /// </summary>
        public string Exhibition_Picture { get; set; }

        /// <summary>
        /// 展覽地點
        /// </summary>
        public string Exhibition_Place { get; set; }

        /// <summary>
        /// 展覽開始時間
        /// </summary>
        public string Exhibition_Startdatetime { get; set; }

        /// <summary>
        /// 展覽標題
        /// </summary>
        public string Exhibition_Title { get; set; }

        public void PK_Exhibition(string Exhibition_ID)
        {

        }
    }
}

File: MODERN-ART-ASSOCIATION-OF-KAOHSIUNG/Scripts/Backstage/EX/Exhibition.js
$(document).ready(function () {
    $.ajax({
        url: '/BEX/GetExhibitiondata_B',
        type: 'POST',
        dataType: 'json',
        success: function (data) {
            console.log(data);
        },
        error: function () {
            console.log("無法取得db");
        }
    });
    //$.ajax({
    //    url: '/BEX/CreateExhibitiondata_B',
    //    type: 'POST',
    //    data: {
    //        Exhibition_Title: 'test333',
    //        Exhibition_Startdatetime: '2015/08/30',
    //        Exhibition_Enddatetime: '2015/09/04',
    //        Exhibition_Place: '暫無資料',
    //        Exhibition_Detail: 'test',
    //        Exhibition_Picture: 'test11',
    //        Exhibition_Otherpeople:'george'
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
    //$.ajax({
    //    url: '/BEX/UpdateExhibitiondata_B',
    //    type: 'POST',
    //    data: {
    //        Exhibition_Title: 'test444',
    //        Exhibition_Startdatetime: '2015/08/31',
    //        Exhibition_Enddatetime: '2015/09/03',
    //        Exhibition_Place: '暫無資料11',
    //        Exhibition_Detail: 'test222',
    //        Exhibition_Picture: 'test11345',
    //        Exhibition_Otherpeople:'georgexkeven',
    //        Exhibition_ID: 49
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
    //$.ajax({
    //    url: '/BEX/DeleteExhibitiondata_B',
    //    type: 'POST',
    //    data: {
    //        Exhibition_ID: 49
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
})